README.md

```diff
@@ -1,26 +1,26 @@
 # GCN Schema
 
-This repository contains the schema for GCN Notices as distributed by [GCN Kafka](https://gcn.nasa.gov). Schema are one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).
+This repository contains the schemas necessary for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). These schemas are one of the important step in the process of establishing [New Notice](https://gcn.nasa.gov/docs/producers) for both existing and new missions.
```

Comment:
"These schema are one of the important step": grammatical error, number agreement. Also, this wording begs the question what the _other_ important steps are.

"process of establishing New Notice for both existing and new missions": "New" should not be capitalized. Also, it looks like there is a missing word here.

Reply:
```suggestion
This repository contains the schemas necessary for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). The formulation of mission schemas are one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).
```